monailabel/scribbles/transforms.py

```diff
@@ -76,6 +76,27 @@ def _copy_affine(self, d, src, dst):
 
         return d
 
+    def _set_scribbles_idx_from_labelinfo(self, d):
+        def convert_string(in_val):
+            if isinstance(in_val, str):
+                # OHIF format - given as "1+idx" string - extract idx
```

Comment:
this should go into client side.. server should have only one standard format.. anything specific let it happen in the client side.. and send the required label info

Reply:
alright! will update this to a single standard format (int) on server and conversions on client-side.